### PR TITLE
Remove inconsistency in translation strings

### PR DESCRIPTION
**Pull request by likhithanimma1**
## Proposed changes

This PR mainly addresses the issue #2935 to remove inconsistency in translation strings

## Release Notes

Milestone:

Changelog:

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (non-breaking change which adds or improves functionality)
- [ ] Breaking change (a change that would cause existing functionality to not work as expected)
- [ ] Documentation (Markdown, README updates)
- [ ] Other (please specify above in "Proposed changes" section)

## Checklist

**General**

- [ ] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/zowe-explorer-vscode/wiki/Best-Practices:-Contributor-Guidance) wiki
- [ ] All PR dependencies have been merged and published (if applicable)
- [ ] A GIF or screenshot is included in the PR for visual changes
- [ ] The pre-publish command has been executed:
  - **v2 and below:** `yarn workspace vscode-extension-for-zowe vscode:prepublish`
  - **v3:** `pnpm --filter vscode-extension-for-zowe vscode:prepublish`

**Code coverage**

- [ ] There is coverage for the code that I have added
- [ ] I have added new test cases and they are passing
- [ ] I have manually tested the changes

**Deployment**

- [ ] I have added developer documentation (if applicable)
- [ ] Documentation should be added to Zowe Docs
  - If you're an outside contributor, please post in the [#zowe-doc Slack channel](https://openmainframeproject.slack.com/archives/CC961JYMQ) to coordinate documentation.
  - Otherwise, please check with the rest of the squad about any needed documentation before merging.
- [ ] These changes may need ported to the appropriate branches (list here):

### CODE REVIEW

**Comment by anaxceron** [comment]
Hi @likhithanimma1 -- I was just curious about how we decided what to capitalize and what to not capitalize. I'd like to make sure we're capitalizing proper nouns or key concepts only.

File: packages/zowe-explorer/l10n/bundle.l10n.json [hunk 1]
```diff
@@ -32,14 +32,14 @@
...
-  "File properties": "File properties",
+  "File Properties": "File Properties",
   "Tag": "Tag",
   "Owner": "Owner",
   "Apply changes": "Apply changes",
   "The API does not support updating attributes for this": "The API does not support updating attributes for this",
   "Waiting for data from extension...": "Waiting for data from extension...",
   "Log in to Authentication Service": "Log in to Authentication Service",
-  "Select a certificate and certificate key in PEM format:": "Select a certificate and certificate key in PEM format:",
+  "Select a certificate and Certificate Key in PEM format:": "Select a certificate and Certificate Key in PEM format:",
```
Comment (zFernand0) [inline packages/zowe-explorer/l10n/bundle.l10n.json:42]: Curious why we capitalize Certificate Key and not Certificate 😋

File: packages/zowe-explorer/l10n/poeditor.json
Comment (zFernand0) [inline packages/zowe-explorer/l10n/poeditor.json:1]: I'm not sure I'm sold on capitalizing each word in the strings, considering that we often don't capitalize every word in a header or similar short-form text (input areas, help-text) that make their way to end-users.

**Review by zFernand0 — COMMENTED** [review]
The PR LGTM as-is (IFF that's what we want to do) 😋 

I'd love to get someone from the Docs Squad to review this PR just to confirm that we are being consistent across Zowe 😅

**Review by JillieBeanSim — DISMISSED** [review]

**Review by t1m0thyj — APPROVED** [review]
LGTM, thanks @likhithanimma1 for making the ZE interface more consistent! 🎉 

After this PR is merged, I can upload the new automatic translations from Artifactory to POEditor 😋

**Review by anaxceron — APPROVED** [review]

**Review by t1m0thyj — APPROVED** [review]

**Review by JillieBeanSim — APPROVED** [review]

**Pull request merged**